apps/chat/channels.py

```diff
@@ -262,29 +264,32 @@ def _participant_is_allowed(self):
     def _new_user_message(self, message) -> str:
         try:
             self._add_message(message)
-        except ParticipantNotAllowedException:
-            return self.send_message_to_user("Sorry, you are not allowed to chat to this bot")
-
-        if not self.is_message_type_supported():
-            return self._handle_unsupported_message()
 
-        if self.experiment_channel.platform != ChannelPlatform.WEB:
-            if self._is_reset_conversation_request():
-                # Webchats' statuses are updated through an "external" flow
-                return ""
+            if not self.is_message_type_supported():
```

Comment:
The diff view is confusing here. I moved the code under the `except ParticipantNotAllowedException` into the `try` block and added another `except  Exception` after that.